hkm/templates/hkm/views/_basket_content.html

```diff
@@ -40,7 +40,7 @@ <h3 class="basket-quantity-value">{% trans "kpl" %}</h3>
     <div class="basket-footer-row">
         <div class="basket-footer-input-wrapper">
             <label>{% trans "Discount code" %}:&nbsp;</label>
-            <input id="discount-code" name="discount_code" type="text">
+            <input aria-label="{% trans 'Discount code' %}" id="discount-code" name="discount_code" type="text">
```

Comment:
Why not connect the label with the input with `for`?

Reply:
Changed now, seems that 7.00 am was too early for my brain to work 😄